src/data/faq.json

```diff
@@ -469,7 +469,8 @@
                         "anchor": "test_in_other_country",
                         "active": true,
                         "textblock": [
-                            "Unfortunately not. Every positive test result that you enter in the app must be verified by labs before. The labs make sure that the result is not entered wrongly or on false pretense. At the moment, only labs in Germany are connected for verification. Therefore, you cannot enter a positive result that was tested in another country, for example, the Netherlands."
+                            "You can only enter a positive test result in Germany's Corona Warn App if the test was taken in Germany. A positive test result received from a laboratory in another country must be entered in the country's corresponding contact tracing app. This ensures that every result is verified, and no result can be entered wrongly or under false pretense. For countries located in the EU, the EU Commission provides a <a href='https://ec.europa.eu/info/live-work-travel-eu/health/coronavirus-response/travel-during-coronavirus-pandemic/mobile-contact-tracing-apps-eu-member-states_en'>list of all European apps</a>.",
```

Comment:
"countries located in the EU" -> "member states of the EU"

Reply:
Changed to "For EU Member States" which is capitalized according to the style guide https://publications.europa.eu/code/en/en-4100400en.htm